### PR TITLE
feat(plugin-meetings) - spark-151331 - video layout switch

### PR DESCRIPTION
**Pull request by 1231varun**
- added an array of layout types accepted by media servers
- added function that accepts layoutType to switch video layout
- added a put request to /controls API to change layout to the specified layout type
- added unit tests

# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Test Coverage

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* SDK Version
* Node/Browser Version
* NPM Version

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/node_modules/@webex/plugin-meetings/src/meeting/index.js [hunk 1]
```diff
@@ -2944,6 +2945,44 @@ export default class Meeting extends StatelessWebexPlugin {
...
+            locusUrl: this.locusInfo.self.url,
+            deviceUrl: this.deviceUrl,
+            layoutType
+          })
+          .then((response) => {
+            if (response && response.body && response.body.locus) {
+              this.locusInfo.onFullLocus(response.body.locus);
+            }
+
+            return Promise.resolve();
```
Comment (mwegman) [inline packages/node_modules/@webex/plugin-meetings/src/meeting/index.js:2976]: This call isn't necessary. You're already in a Promise chain

File: package.json [hunk 2]
```diff
@@ -229,7 +229,7 @@
     "urlsafe-base64": "^1.0.0",
     "uuid": "^3.3.2",
     "valid-url": "^1.0.9",
-    "wd": "^1.11.3",
```
Comment (mwegman) [inline package.json:232]: Why are you updating these packages?

Comment (1231varun) [reply package.json:232]: I didn't update these, all I did was npm install.

File: packages/node_modules/@webex/plugin-meetings/src/meeting/index.js (hunk 1 above)
Comment (1231varun) [reply packages/node_modules/@webex/plugin-meetings/src/meeting/index.js]: Addressed!

File: package.json (hunk 2 above)
Comment (1231varun) [reply package.json]: Reverted both package.json and package.lock.json files

**Pull request merged**